src/proj.rs

```diff
@@ -584,7 +585,7 @@ impl Proj {
     pub fn project<C, F>(&self, point: C, inverse: bool) -> Result<C, ProjError>
     where
         C: Coord<F>,
-        F: Float,
+        F: Float + Debug,
```

Comment:
Is there a reason proj::CordinateType isn't Float?

Then this could be:
F: proj::CordinateType

Reply:
To be clear, this is fine as is. 

The approach I've been taking elsewhere is to try to avoid specifying num_traits in many places, and centralize it's use in something like geo_types::CoordNum/CoordFloat and geo::GeoNum/GeoFloat.

(there are still several exceptions where we're refining on more specific num_traits throughtout geo because I didn't want to start a riot by precluding people who's exotic numeric type doesn't implement... negative...)

Reply:
OK, now with `Float` bound on `CoordinateType` and simpler bound on `F`. I like this better.